src/XmlReader.cpp

```
@@ -1,47 +1,70 @@
+		if (ec != std::errc() || ptr != end)
 			throw std::string("not a float value");
 		return ret;
 	}
 
 	static inline int convertInt(const std::string& str)
 	{
-		auto begin = str.c_str();
-		char* end;
+		std::string_view sv = trimAsciiWhitespace(str);
+		// strtol accepted a leading '+'; from_chars doesn't, so skip it.
+		if (!sv.empty() && sv.front() == '+')
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)